File: workspace/people/p02-site-reliability-engineer/home/documents/operations/runbooks/automation/2026-07-20/source/2026-07-20-reliability-runbook-node-pressure-automation.py
"""Northstar Transit Cloud; Checkout SVC-17; 2026-07-13; error budget 2.4%; INC-260713."""
SERVICE = "Checkout SVC-17"
INCIDENT = "INC-260713"
TOPIC = "node_pressure"
SPINE_IDS = ['org.ntc', 'svc.checkout.svc17', 'rel.inc260713.timeline', 'date.2026-07-20', 'term.no-kio-claim']
FINGERPRINT = "91533f26da75cf0e"

def capacity_decision(headroom_percent: int) -> str:
    return 'hold' if headroom_percent >= 31 else 'escalate'

File: workspace/people/p02-site-reliability-engineer/home/documents/operations/runbooks/automation/2026-07-20/source/2026-07-20-reliability-runbook-node-pressure-automation.rs
// Northstar Transit Cloud | Checkout SVC-17 | 2026-07-13 | error budget 2.4% | INC-260713
pub const TOPIC: &str = "node_pressure";
pub const FINGERPRINT: &str = "cc572c6116b3c67a";
pub fn headroom_ok(percent: u8) -> bool { percent >= 31 }
